source/clear-linux/guides/maintenance/swupd-mixer-integration.rst

```diff
@@ -0,0 +1,117 @@
+.. _swupd_mixer_integration:
+
+Adding custom packages and bundles
+##################################
+
+
+
+Prerequisites
+=============
+
+To use the new swupd mixer feature, you'll need a recent image of Clear
+Linux OS for Intel Architecture with the following bundle installed. If you
+don't have it already, you can add it with the :command:`swupd bundle-add`
+command like so::
+
+  # swupd bundle-add mixer
```

Comment:
> Prerequisites
 > =============
 > 
 > To use the new swupd mixer feature, you'll need a recent image of Clear
 > Linux OS for Intel Architecture with the following bundle installed. If you
 > don't have it already, you can add it with the :command:`swupd bundle-add`
 > command like so::
 >
 >  # swupd bundle-add mixer

Recommend putting this at the beginning of the "How to use" section instead.  The beginning of this doc. should be "What is it"